src/components/ShoppingCart/CartListItem.tsx

```diff
@@ -5,6 +5,7 @@ import { MdDeleteForever } from "react-icons/md";
 import { ICartItem } from "@/interfaces/cartItem";
 import DropDownButton from "@/components/ShoppingCart/dropDownButton";
 import { RiErrorWarningLine } from "react-icons/ri";
+import Image from "next/image";
```

Comment:
As I said before, It is not going to work unless we change our production environment to SSR. Talk to our Devops about it. 
There are 2 documents on how to deploy Nextjs into AWS with SSR support.
https://aws.amazon.com/about-aws/whats-new/2021/08/aws-amplify-hosting-support-next-js-version-11/
https://docs.amplify.aws/guides/hosting/nextjs/q/platform/js/